src/app/components/toolbar/index.html

```diff
@@ -8,6 +8,15 @@
     The toolbar component displays a SKY UX-themed toolbar. Use the <stache-code>sky-toolbar</stache-code> component in conjunction with the <stache-code>sky-toolbar-item</stache-code> child component.
   </sky-demo-page-summary>
 
+  <sky-demo-page-properties sectionHeading="Toolbar component">
```

Comment:
I don't think this should be a new section. I think `sky-toolbar-view-actions` is another child component to add to the list in the existing Toolbar child components section. @Blackbaud-TrevorBurch, is that correct, or am I missing something?

Otherwise, this LGTM. If Trevor agrees, I think we just need to move this `sky-demo-page-property` subsection to the other section.

Reply:
You are right @blackbaud-johnly. That is my mistake. I told @Blackbaud-DeniseCisneros that it was wrong earlier but I misread the stache code.... It should go alongside `sky-toolbar-item` and `sky-toolbar-section` in the `Toolbar child components` section.

Sorry about that @Blackbaud-DeniseCisneros!

Reply:
No problem! Fixed :)